src/VolleyManagement.Backend/VolleyManagement.Services/TournamentService.cs

```diff
@@ -168,7 +168,9 @@ public List<Tournament> GetFinished()
         /// <returns>Tournament teams</returns>
         public List<TeamTournamentDto> GetAllTournamentTeams(int tournamentId)
         {
-            return _tournamentTeamsQuery.Execute(new FindByTournamentIdCriteria { TournamentId = tournamentId });
+            var result = _tournamentTeamsQuery.Execute(new FindByTournamentIdCriteria { TournamentId = tournamentId });
+
```

Comment:
What is the reason you added this variable?

Reply:
It was me to show lack of UT. But it should be reverted.